docs/guides/guides/network-requests.mdx

```diff
@@ -541,6 +542,17 @@ test in the Command Log. Here is an example of what this looks like:
   alt="Screenshot of fetch and XHR requests"
 />
 
+:::info
+
+**Note:** Logging can be disabled off by passing `{ log: false }` with your
+static response.
+
+```js
+cy.intercept('/users*', { body: ['user1', 'user2'], log: false }).as('getUsers')
```

Comment:
Should we use the example that disables all xhr/fetch logs?

https://docs.cypress.io/api/commands/intercept#Disabling-logs-for-a-request

Reply:
I specifically used the example from farther down in the code to match: https://github.com/cypress-io/cypress-documentation/pull/5124/files#diff-bb8ce895b7660f9f324902a3e98c85d05f942fdd16997394dd5fc0dc46abb095L554

but could change to disable all if people want.

Reply:
Could also link out to the example that shows how to disable all if we didn't want to update here.

Reply:
updated in c020484